handling-forms/problem-set-forms-and-apis.md

```diff
@@ -31,6 +55,48 @@ The web app should also display:
 - The latitudinal coordinate of the search result
 - The longitudinal coordinate of the search result
 
+## Resources
+
+Scaffold code is available to help you get started quickly.
+
+- [Forms and APIs Problem Set](https://github.com/AdaGold/problem-set-forms-and-apis)
+
+After forking and cloning the project, run `npm install` to install the necessary dependencies. The development server is started with `npm run dev`.
+
+The scaffold code removes a lot of the default Vite template code and simplifies the applied styles. The `App` component renders an empty body.
+
+A `.env.sample` file is provided that can be used to store the API key. Make a copy of the file and name it `.env`. Then update the LocationIQ key in that file. The `.env` file is already added to the `.gitignore` file, so it will not be committed to the repository. We should still avoid attempting to deploy this to any public location, as the API key will still be visible in the source code.
```

Comment:
Resources also came from an earlier problem set, but it presented the starting files as summary code blocks that the reader needed to copy and paste. I can remove this part and merge with the above section.

I'm not sure what the best approach with the .env file is here, since I don't think we actually discuss the React-style .env anywhere, but I used it in the sample. I think it's good to help students avoid checking in their key, but maybe something needs to be written about that?

This might be easier to discuss in more detail offline.

Reply:
After condensing the other sections (Practice and Resources) we'll add a background section about React (Vite) .env files before students get to the Implement section.